polly-merge.py

```diff
@@ -235,12 +235,13 @@ def process_pr(pr_data, bitbucket_url, auth_header, merge_trigger):
     projectkey = pr_data["toRef"]["repository"]["project"]["key"]
     pullrequestid = pr_data["id"]
 
-    all_comments = get_all_comments(
-        bitbucket_url, auth_header, projectkey, repositoryslug, pullrequestid,
-    )
+    def get_comments():
+        return get_all_comments(
+            bitbucket_url, auth_header, projectkey, repositoryslug, pullrequestid
+        )
 
-    # look for exact match in any comment
-    if merge_trigger in all_comments:
+    # look for exact match in PR description or any comment
+    if merge_trigger in pr_data["description"] or merge_trigger in get_comments():
```

Comment:
hmm, I should have tested this, seems to fail for me? verifying.

Reply:
Fixed in e4d943b45b082927d287e1e4e008214e87f14a1f .

Reply:
doh nice catch. RIIR